scripts/gen_coveragefile_ghidra.py

```diff
@@ -0,0 +1,29 @@
+#Output a module coverage file for wtf to %USERPROFILE%/ghidra_scripts
+#@author huwwp
+#@category fuzzing
+
+from ghidra.program.model.block import BasicBlockModel
+import json
+import os
+
+program = getCurrentProgram()
+program_name = program.getName().rsplit(".",1)[0]
+base_address = program.getImageBase().getOffset()
+block_iterator = BasicBlockModel(program).getCodeBlocks(monitor)
+block = block_iterator.next()
+
+address_list = []
+
+while block:
+    address_list.append(block.minAddress.getOffset() - base_address)
+    block = block_iterator.next()
+
+json_object = {
+	'name': program_name,
+	'addresses': address_list
+}
+
+file = open(os.environ['USERPROFILE'] + "/ghidra_scripts/" + program_name + ".cov", "w")
```

Comment:
does it make sense to try to output the file next to where the currently loaded file is on the system? That's what the IDA version of the script does.

Reply:
Yeah makes sense, updated to same behaviour